core/templates/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -24,7 +24,7 @@ import { HtmlEscaperService } from 'services/html-escaper.service';
 interface RteComponentSpecs {
   backendId: string;
   customizationArgSpecs: {
-    name: string, value: unknown, 'default_value': unknown
+    name: string; value: unknown; 'default_value': unknown;
```

Comment:
Let's maybe put these on separate lines, just to be consistent throughout the codebase. @vojtechjelinek @DubeySandeep wdyt?

Reply:
I'm indifferent about this, I think that if it fits on one line it can stay on one line.

Reply:
I don't have any strong opinion on this but I prefer having one item in line (`;` looks kinda statement end indicator, so following one statement in a line seems cleaner).